user wallet update - missing screenshots (#134)
exoplatform/exo-documentation: docs/UserWallet.rst

|image1|

How to create your wallet
~~~~~~~~~~~~~~~~~~~~~~~~~~

The first thing to do is to create your wallet by setting a password. The password must contain at least 8 characters. 

|image3|

> Once your password is set, your wallet is created but you will not be allowed to make transactions until an administrator approves it. The wallet's interface is nevertheless available.

Reviewer: ```suggestion
Once your password is set, your wallet is created but you will not be allowed to execute transactions (such as purchases in the store or sending tokens fro. the wallet) until an administrator approves it.  Nevertheless, the The wallet interface remains available.
```

> Once your password is set, your wallet is created but you will not be allowed to execute transactions (such as purchases in the store or sending tokens from the wallet) until an administrator approves it.  Nevertheless, the The wallet interface remains available.

Reviewer: ```suggestion
Once your password is set, your wallet is created but you will not be allowed to execute transactions (such as purchases in the store or sending tokens from the wallet) until an administrator approves it.  Nevertheless, the wallet interface remains available.
```

|image5|

Once your wallet is approved by an administrator, you’ll receive a notification informing you that your wallet is activated.

|image|

All actions become available and you can send and receive funds as well as view transaction history or statistics and reward details.

|image|

Wallet security
~~~~~~~~~~~~~~~~

Backup your wallet
------------------

> Each wallet uses a pair of cryptographic keys (strings of characters) : the public key and the private key. The public key represents the address of your wallet on the blockchain network. The private allows works with the public key but must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's critical to keep your private key in a safe place (such as multiple paper copies if different physical locations) to protect it from loss or theft :

Reviewer: ```suggestion
Each wallet uses a pair of cryptographic keys (strings of characters) : the public key and the private key. The public key represents the address of your wallet on the blockchain network. The private works with the public key to encrypt/decrypt data securely.   ut it must be secured as it allows to perform any operation with the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's critical to keep your private key in a safe place (such as multiple paper copies if different physical locations) to protect it from loss or theft :
```

|image5|

> Once your wallet is approved by an administrator, you recieve a notification informing you that your wallet is activated.

Reviewer: ```suggestion
Once your wallet is approved by an administrator, you’ll receive a notification informing you that your wallet is activated.
```

|image|

> All actions become available and you are allowed to send and recieve funds as well as consulting the transactions hoistory, rewarding details and see the transactions statistics.

Reviewer: ```suggestion
All actions become available and you can send and receive funds as well as view transaction history or statistics and reward details.
```

> It is possible to filter ...

Reviewer: @samehaidi incomplete?

Author: I wanted to add a paragraph that is related to statistics capabilities for filters. 
I think it is not the right location to put it.
WDYT?

Reviewer: I don’t knowif it is the right location but it can’t stay like this.

Author: i deleted it.

|image|

Wallet security
~~~~~~~~~~~~~~~~

Backup your wallet
------------------

> Each wallet uses a pair of public and private keys. The public key is used as the address on the blockchain network. The private key must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's important to secure the private key and save it in case of loss  :

Reviewer: ```suggestion
Each wallet uses a pair of cryptographic keys (strings of characters) : the public key and the private key. The public key represents the address of your wallet on the blockchain network. The private allows works with the public key but must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's critical to keep your private key in a safe place (such as multiple paper copies if different physical locations) to protect it from loss or theft :
```

> Each wallet uses a pair of public and private keys. The public key is used as the address on the blockchain network. The private key must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's important to secure the private key and save in case of loss  :

Reviewer: ```suggestion
Each wallet uses a pair of public and private keys. The public key is used as the address on the blockchain network. The private key must be secured as it allows to do any operation in the blockchain account (such as stealing your funds!). Without the private key, access to the funds is impossible. Thus, it's important to secure the private key and save it in case of loss  :
```

.. Warning:: Remember that losing your private keys means losing all your crypto-money. You must take precautions and be very careful! Even an administrator won't be able to help you if it gets compromised or lost.

Wallet password
----------------

You can change your wallet password by clicking on "Change password" button from security popup. You just need to enter your current password and the new password then confirm.

|image20|

The option "Remember my password on this browser" stores your password in the browser’s local storage to be automatically used for every transaction (including purchases in the Perk Store) without requiring your input each time. 

|image21|

You must enter your current password to activate this option.

|image22|

How to send funds from wallet
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

To send funds from your wallet, you have to click on "Send" button then, in the "send funds" popup, select a recipient, your wallet password and the amount to send. You can optionally type a label and a message that will be stores in the transaction history.
You also have to choose the type of transaction between three options: "Cheap", "Normal" or "Fast". 

> The difference between those types of transaction is the price and the speed of processing transaction in the blockchain network. There is no guaranteed delays for transaction processing because it depends on how much the blockchain network is busy, but basically, the more you pay, the faster. Below are some indicative delays to expect :

Reviewer: ```suggestion
The difference between these transaction modes is the fees and the speed of processing transaction in the blockchain network. There is no guaranteed delay for transaction processing because it depends on how much the blockchain network is busy. But basically, the more you pay, the faster. Below are some indicative delays to expect :
```

 
- The cheap mode is not expensive but may take up to one day to be executed.

> - The recommended mode is slightly more expensive and could take few hours.

Reviewer: ```suggestion
- The recommended mode is slightly more expensive and may take a few hours.
```

> - The fast most is the most expensive and usually takes only one minute to be executed.

Reviewer: ```suggestion
- The fast mode is the most expensive but transactions are usually processed in less than a minute.
```

.. Warning:: Remember that losing your private keys means losing all your crypto-money. You must take precautions and be very careful! Even an administrator won't be able to help you if it gets compromised or lost.

Wallet password
----------------

> You can change your wallet password by clicking on "Change password" button from security popup. You just have to type your current password and the new password and confirm the action.

Reviewer: ```suggestion
You can change your wallet password by clicking on "Change password" button from security popup. You just need to enter your current password and the new password then confirm.
```

|image20|

> The option "Remember my password on this browser" allow you to save the wallet's password in the browser so you don't have to write it for every transaction and when you buy products from perkstore using the browser in question. 

Reviewer: ```suggestion
The option "Remember my password on this browser" stores your password in the browser’s local storage to be automatically used for every transaction (including purchases in the Perk Store)without requiring your input each time. 
```

|image21|

> You have to type your actual password to activate this option.

Reviewer: ```suggestion
You must enter your current password to activate this option.
```

> The difference between those types of transaction is the price and the rapidity of transaction:

Reviewer: ```suggestion
The difference between those types of transaction is the price and the speed of processing transaction in the blockchain network. There is no guaranteed delays for transaction processing because it depends on how much the blockchain network is busy, but basically, the more you pay, the faster. Below are some indicative delays to expect :
```

> - The cheap type is not expensive but may take up to one day to be executed.

Reviewer: ```suggestion
- The cheap mode is not expensive but may take up to one day to be executed.
```

> - The recommended type is slightly more expensive and could take few hours.

Reviewer: ```suggestion
- The recommended mode is slightly more expensive and could take few hours.
```

> - The fast type is the most expensive and take only one minute to be executed.

Reviewer: ```suggestion
- The fast most is the most expensive and usually takes only one minute to be executed.
```

|image12|

When you click on the plus icon next to the total rewarded, a drawer with all rewards details is displayed.

|image18|

> You can see more information about a reward transaction by clicking in the corresponding line. The information about the number of gamification points earned and kudos recieved during the period of reward.

Reviewer: ```suggestion
You can see more information about a reward transaction by clicking in the corresponding line. The information about the number of gamification points earned and kudos received during the period of reward.
```

|image19|

Tansactions details
~~~~~~~~~~~~~~~~~~~~

The amount of the last transaction is displayed in the wallet interface.

|image13|

When you click on the plus icon next to the last transaction information, a drawer with all transactions is displayed.

|image14|

To see a transaction details (its status, label, message, amount, the sender and recipient adresses, the contact name and address, the transaction fee and hash) just click on the pull down arrow from the transaction list. 

|image15|

> In the case of a failed transaction, a warning is displayed in the transaction list and the transaction details. The arrow and transaction amount are displayed in red color so that you can distingish them.

Reviewer: ```suggestion
In case of a failed transaction, a warning is displayed in the transaction list and the transaction details. The arrow and transaction amount are displayed in red color so that you can distingish them.
```
